src/screens/Login/Login.tsx

```diff
@@ -132,28 +81,36 @@ const Login = () => {
         </Text>
       </View>
 
+      <View style={[GlobalStyles.mb20]}>
+         <Text style={[
+          GlobalStyles.textRed
+         ]}>{errorReducer.message}</Text>
```

Comment:
You need to add the font-family as well to the text else it will look different from others

Reply:
FOnt family added. thanks

Reply:
I think since you are making the UI adjustment here, it would be better if the text was centralized and the `font-size ` smaller probably 13px would be fine

Reply:
Centralizing it will not obey the text's visual hierarchy. but if you think it should be so...then I do it. what do you think?